src/policy_evaluator.rs

```diff
@@ -268,6 +215,8 @@ impl PolicyEvaluator {
 mod tests {
     use super::*;
 
+    use std::collections::HashMap;
```

Comment:
Is that really needed, I don't see any change below compared to the original tests? :thinking:

Reply:
The `HashMap` itself of the Wapc mapping got moved to another mod. However, if I keep the `use` I get warnings of an unused import, because in this file, the `HashMap` is only used in `test` build mode. This is why the usage has to go inside the conditional compilation, so it builds and runs the unit tests, and it also does not print any warning about an unused import.

Reply:
Thanks!